estate_account/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+from odoo import models, api, fields
+class EstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    #------------------------------------------Action Methods----------------------------------------------
+    def property_sold_action(self):
+        """set property state as sold and Create invoice for sold property"""
+        res = super().property_sold_action()
+        journal = self.env["account.journal"].search([("type", "=", "sale")], limit=1)
+        for property in self:
+            self.env["account.move"].create({
+                'partner_id': property.buyer_id.id,
+                'move_type': 'out_invoice',
+                "journal_id": journal.id,
+                'invoice_date': fields.Date.today(),
+                "invoice_line_ids": [
+                    (0,0,{
+                            "name": property.name,
+                            "quantity": 1.0,
+                            "price_unit": property.selling_price * 6.0 / 100.0,
```

Comment:
```suggestion
                            "price_unit": property.selling_price,
```
why the property line just have 6% of the amount at which it was sold?

Reply:
sir , as per the excercise  "Add the third step of invoice creation.
Add two invoice lines during the creation of the account.move. Each property sold will be invoiced following these conditions:
6% of the selling price
an additional 100.00 from administrative fees
"
I am unsure about whether  it is supposed to be 6% extra on the selling price or is it supposed to be 6% on the selling price